packages/openapi-generator/src/file-serializer/docs.ts

```diff
@@ -0,0 +1,83 @@
+import { EOL } from 'os';
+import { documentationBlock } from '@sap-cloud-sdk/util';
+import type {
+  OpenApiApi,
+  OpenApiNamedSchema,
+  OpenApiObjectSchemaProperty,
+  OpenApiOperation,
+  OpenApiParameter,
+  OpenApiRequestBody
+} from '../openapi-types';
+
+export function apiDocumentation(api: OpenApiApi, serviceName: string): string {
```

Comment:
Rather minor: I am not a fan of the additional `doc.ts` file as I don't see much shared functionality/reuse between/of the functions in here. It also contradicts the existing structure a little. I believe that all the functions here will be called from one module each, so maybe we should we just put it there, close to where it is needed?

Reply:
I agree totally. However, It is nice if you have the tests for the docs in one file because it is easier to see them all. I will refactor.

Reply:
So I would keep the docs.spec.ts?